pkg/querier/tripperware/instantquery/InstantQueryMiddlewares.go

```diff
@@ -0,0 +1,32 @@
+package instantquery
+
+import (
+	"flag"
+
+	"github.com/go-kit/log"
+
+	"github.com/cortexproject/cortex/pkg/querier/tripperware"
+)
+
+// Config for query_range middleware chain.
```

Comment:
I thought this is for instant query?

Reply:
yeah... will have to polish the naming on this PR